mods/default/init.lua

```diff
@@ -17,7 +17,7 @@ default.get_translator = S
 -- and avoids obscure, hard to debug runtime errors.
 -- This section should be updated before release and older checks can be dropped
 -- when newer ones are introduced.
-if ItemStack("").add_wear_by_uses == nil then
+if not minetest.features.node_interaction_actor then
```

Comment:
I can see that it's used but would it stop working without this feature? I don't think so.

Reply:
I have to know whether the node digging was successful. However, `minetest.node_dig` returns no value/does not have documentation on its return value.

Reply:
Ah so it's about the return value, right.